### PR TITLE
Added react-home page

### PR DESCRIPTION
**Pull request by ashuydv**
Made Home Page with Reactjs,

### CODE REVIEW

**Comment by ashuydv** [comment]
The deployment have failed, what should I do ??

File: README.md [hunk 1]
```diff
@@ -1,159 +1,70 @@
-# Awesome Portfolio Websites🌐
```
Comment (anushbhatia) [inline README.md:1]: don't modify the readme.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (anushbhatia) [inline .vscode/settings.json:1]: Remove this file  too.

File: src/setupTests.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (anushbhatia) [inline src/setupTests.js:1]: Remove these comments too

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Keep fonts in the assets folder too.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
+    "name": "awesome-portfolio-webs",
```
Comment (smaranjitghose) [inline package.json:2]: Put the proper name please and update the remaining files!

File: src/components/Education/Education.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Education = () => {
+    return (
+        <div>
+            <h1>Hello, Education</h1>
```
Comment (smaranjitghose) [inline src/components/Education/Education.js:6]: Reorganize these into pages folder.

[Reference](https://github.com/smaranjitghose/doc2pen/tree/master/src/pages)

File: README.md (hunk 1 above)
Comment (ashuydv) [reply README.md:1]: okk, will do

**Comment by ashuydv** [comment]
ADDED RESEARCH PAGE,

File: src/components/Education/Education.js (hunk 5 above)
Comment (ashuydv) [reply src/components/Education/Education.js:6]: Will resolve tommorow,as I've to change all the files and thier names and paths

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**